docs/v3/source/includes/resources/apps/_clear_buildpack_cache.md.erb

```diff
@@ -0,0 +1,36 @@
+### Clear buildpack cache for application
+
+```
+Example Request
+```
+
+```shell
+curl "https://api.example.org/v3/apps/[guid]/actions/clear_buildpack_cache" \
+  -X POST \
+  -H "Authorization: bearer [token]"
+```
+
+```
+Example Response
+```
+
+```http
+HTTP/1.1 202 Accepted
+Location: https://api.example.org/v3/jobs/[guid]
+```
+
+This endpoint will delete all of the existing buildpack caches for a specified app.
```

Comment:
This endpoint will delete the buildpack cache for a specified app.
(there only one cache per app)